src/layers/feedforward.rs

```diff
@@ -1,7 +1,5 @@
-use candle_core::{Module, Tensor};
-use candle_nn::{linear, linear_no_bias, Linear, VarBuilder};
-
-use crate::error::Result;
+use candle_core::{Module, ModuleT, Tensor};
+use candle_nn::{linear, linear_no_bias, Dropout, Linear, VarBuilder};
```

Comment:
No errors for this module?

Reply:
The issue is that this module implements `ModuleT` and we cannot implement that with a custom error type. I think in a follow-up PR, we should define our own module trait that is more flexible.